src/chat_analyzer.py

```diff
@@ -230,11 +231,72 @@ def controller(
             plt.show()
 
     if activity:
-        check_activity(msgs, username, start_date, end_date, show_graph)
+        if username:
+            list_activity, list_graph = check_activity(msgs, username, start_date, end_date, show_graph)
+
+            print('The user {} mostly stays active around {} Hours'.format(list_activity[0], list_activity[1]))
+
+            if show_graph and globals.CAN_SHOW_GRAPH:
+                print('\nShowing graph....')
+                plt.plot(list_graph[0], list_graph[1])
+                hours = ['00', '01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11',
+                         '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23']
+                plt.xticks(ticks=np.arange(24), labels=hours)
```

Comment:
The `HOURS_LIST` from globals should be used here like it was [previously being used](https://github.com/JRavi2/Chat-Analyzer/pull/16/files#diff-42db47af89ce6cfd4f4994c4b00c4efdffec946c80f8f26238b40e25339f2200L164).

Reply:
Every time I would run the program, it would return an error stating something around 'globals.HOURS_LIST not defined'